Networking/Networking/Model/OrderItem.swift

```diff
@@ -7,42 +7,45 @@ public struct OrderItem: Decodable {
     public let itemID: Int
     public let name: String
     public let productID: Int
+    public let variationID: Int
```

Comment:
I moved the variationID up here so the model would be a closer representation of the response object's field order.

Reply:
Nice touch!